### ISSUE TITLE
Special episodes recognition bug (S00E01 format) 

### ISSUE THREAD

**Issue by GaussianCurve** [iamkroot/trakt-scrobbler#93]
The scrobbler is working, but for some reason it confuses the specials for anime with the main season.
 
For example it will confuse specials like this: 
https://trakt.tv/shows/the-qwaser-of-stigmata/seasons/0/episodes/2
 
With the episode from the actual season of the show which is this:
https://trakt.tv/shows/the-qwaser-of-stigmata/seasons/1/episodes/2
 
The file name I used for the special episode is this:  The Qwaser of Stigmata - S00E02 - Picture Drama 02 - Secret Episode 1 - Tomo
 
 
The file name I used for the second episode of season one is this: The Qwaser of Stigmata - S01E02 - Friendship of Masks

Sorry I am a bit new to python and using Github so this is probably an easy solution staring me in the face.

**Comment by iamkroot** [comment]
Thanks for reporting the issue! Turns out this is a bug in the episode info extraction logic, and I also noticed this just last week when I was refactoring that part of the code. The reason is that anime generally don't have seasons, and filenames only contain the episode number. In this case, the current behaviour is that if the season is not present, it defaults to 1 (which is what trakt accepts for anime). The bug is that this "season not present" check also counts `S00` kind of files as if the season isn't there.
https://github.com/iamkroot/trakt-scrobbler/blob/908215bb57a96bc7baa7b973441a7f6d9ff11ecb/trakt_scrobbler/file_info.py#L68
(Here, there should be an additional check if the season is 0, since `0 or 1` evaluates to `1` in python)

 I have a fix in my local branch, as a part of better support for anime absolute numbered episodes. Sadly, I'm blocked on a Trakt API issue (https://github.com/trakt/api-help/issues/224), so once that's resolved, I'll push the fix.
